lib/utils.js

```diff
@@ -161,20 +161,20 @@ module.exports.mapToMac = function (options) {
 module.exports.actionJackerDecorator = function (emitter, options, fn, mapper) {
   options = clone(options);
   fn = fn || function (err, data) {};
-  return function (err, data) {
-
-    var resultantData = data;
-    // Sanitize the data
-    if(resultantData) {
-      resultantData = resultantData.toLowerCase().trim();
-      if(resultantData.match(/^activate/)) {
-        resultantData = 'activate';
-      }
-    }
-    fn.apply(emitter, [err, resultantData]);
-    if (err || !mapper || !resultantData) return;
-
-    var key = mapper(resultantData);

```

Comment:
https://github.com/mikaelbr/node-notifier/pull/115/files#diff-50e3aa130a4f97a42ee2cf111c7b1d9dL164
mix EOL here
`git config --global core.safecrlf true`

see: http://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/